src/main/java/org/kohsuke/github/GHLabel.java

```diff
@@ -43,7 +50,23 @@ public void delete() throws IOException {
      *      6-letter hex color code, like "f29513"
      */
     public void setColor(String newColor) throws IOException {
-        repo.root.retrieve().method("PATCH").with("name", name).with("color", newColor).to(url);
+        repo.root.retrieve().method("PATCH")
```

Comment:
Assuming that it is needed seems like a low risk choice.

Reply:
@deadmoose 
If you are still around what do you think about using the builder/updater style similar to Release/ReleaseBuilder/ReleaseUpdater?
That way the original object remain unchanged, but you can still update and do it all in one network call.  This also sidesteps the question of when all fields need to be sent.  

I admit it would be a bit more work to do - We'd want to mark existing `set*` methods a deprecated, but it wouldn't be that bad.